cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -4587,6 +4647,24 @@ TEST_F(TestArrowReadDeltaEncoding, DeltaBinaryPacked) {
   ::arrow::AssertTablesEqual(*actual_table, *expect_table);
 }
 
+TEST_F(TestArrowReadDeltaEncoding, DeltaBinaryPackedWithLargeBinaryVariant) {
```

Comment:
I'm sorry that I was thinking this is for DELTA_LENGTH_BYTE_ARRAY encoding. Please remove this test as it does not relate to BYTE_ARRAY type. Sorry about that again! I will make a follow change to make sure it is covered here.